chess/backend/src/game/logic.py

```diff
@@ -207,17 +208,53 @@ def _convert_SAN_str_to_move(san: str) -> Move:
     move = _get_move(from_coord, to_coord)
     return move
 
+
 def _get_most_recent_move(game_id: int) -> Moves:
     moves = Moves.objects.filter(game_id_id=game_id)  # Django has weird name-mangling
     return moves.order_by("-move_number")[0]
 
+
 def record_move(self, board: Board, move: Move, game_id: int):
     most_recent_move = _get_most_recent_move(game_id)
     move_state = MoveState()
-    move_state.set_state_from_prev_move(most_recent_move)      
+    move_state.set_state_from_prev_move(most_recent_move)
     move_state.post_move_fen = board.asFen()
     move_state.set_move(move)
     add_move_state_to_database(move_state)
-    
+
+
 def add_move_state_to_database(move: MoveState):
-    move = Moves.objects.create(**move.items())
\ No newline at end of file
+    move = Moves.objects.create(**move.items())
+
+
+def get_current_games(request: WSGIRequest) -> JsonResponse:
+    user = get_user_info(request)
+    all_games = Games.objects.filter(Q(user_id_1=user.id) | Q(user_id_2=user.id))
+
+    active_games = all_games.filter(ongoing=1)
+    game_list = []
+    for game in active_games:
+        game_list.append(build_game_status_dict(game, user))
+
+    return JsonResponse({"games": game_list})
+
+
+def build_game_status_dict(game: Games, user: Users) -> Dict[str, str]:
+    most_recent_move = _get_most_recent_move(game.id)
+
+    game_dict = {}
+    game_dict["id"] = game.id
+    game_dict["color"] = WHITE if game.white_user_id == user.id else BLACK
+    game_dict["turn"] = BLACK if most_recent_move.turn == WHITE else WHITE
+    game_dict["count"] = most_recent_move.move_number
+
```

Comment:
maybe:

game_dict = {
   'id': game.id,
   'color': ....,
   ...
}

Reply:
I thought about this, but decided against it. Looking at it again, however, I agree with you. I've updated it.